src/components/learning-log.tsx

```diff
@@ -52,12 +52,17 @@ export class LearningLogComponent extends BaseComponent<IProps, {}> {
           {this.renderWorkspace(
               "left-workspace",
               "primary",
-              <DocumentComponent document={primaryWorkspace} workspace={learningLogWorkspace} side="primary" />
+              <DocumentComponent
+                document={primaryWorkspace}
+                workspace={learningLogWorkspace}
+                isGhostUser={false}
```

Comment:
Seems like there's another concept here that could be more descriptive...e.g. is it editable. Break down into component pieces